CMakeLists.txt

```
@@ -542,6 +542,10 @@ endif()
 set_source(xsimd)
 resolve_dependency(xsimd 10.0.0)
 
+set(stemmer_SOURCE BUNDLED)
+set_source(stemmer)
```

```suggestion
set(stemmer_SOURCE BUNDLED)
```
Looking at the library source they don't seem to provide any form of config or pc file ... so that would require us to create a Findstemmer.cmake. Depending on compile time it might be worth it, if it's fast we can skip it though and always bundle for now.

---

yea, the build is pretty fast. So I leave it as-is.